### PR TITLE
[WIP] Major refactoring

### PR DESCRIPTION
**Pull request by iamakulov**
Closes #120.
A part of #31.

**TO DO** (the list will be updated):
- [ ] Rename `svgi` in the service descriptor into something more meaningful
- [ ] Rename `name` in the service descriptor into something like `publicButtonName`: this parameter goes into the button class, so it should be explicit that it shouldn’t be changed for backward compatibility
- [ ] Replace `utils.js` with Lodash (make sure it doesn’t affect the bundle size)

### CODE REVIEW

File: source/button.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
+import _ from 'lodash';
```
Comment (vitkarpov) [inline source/button.js]: Is full lodash loaded into bundle?

Comment (iamakulov) [inline source/button.js]: Nope, I use [`babel-plugin-lodash`](https://github.com/lodash/babel-plugin-lodash) with [`lodash-webpack-plugin`](https://github.com/lodash/babel-plugin-lodash)

Comment (vitkarpov) [inline source/button.js]: Super cool! 👍

**Comment by iamakulov** [comment]
I’ll close this for now to clean up the PR list. I’m going to come back to this in July

**Pull request closed without merging**